frontend/src/metabase/nav/components/WhatsNewNotification/utils.ts

```diff
@@ -12,13 +12,15 @@ export const getLatestEligibleReleaseNotes = ({
   currentVersion,
   lastAcknowledgedVersion,
   isEmbedded = false,
+  isWhiteLabeling = false,
 }: {
   versionInfo: VersionInfo | null;
   currentVersion?: string;
   lastAcknowledgedVersion: string | null;
   isEmbedded?: boolean;
+  isWhiteLabeling?: boolean;
```

Comment:
`getIsWhiteLabeling` always returns a boolean. Let's make it required, so that it's not accidentally omitted when calling this function.

Probably all parameters in this function should be required for the same reason.
If they can be `undefined`, they should be typed with `| undefined`.

Reply:
Thanks!
TIL that there's a difference between `?: string` and `: string | undefined`